camayoc/tests/qpc/api/v1/scanjobs/test_pause_cancel_restart.py

```diff
@@ -0,0 +1,80 @@
+# coding=utf-8
+"""Tests for  quipucords scans and reports.
+
+These tests are parametrized on the inventory listed in the config file.
```

Comment:
`The tests are parametrized on the QPC_SOURCE_TYPES and will skip if there is not a source of that type in the config file.`?